### PR TITLE
Update docker cli message for case where user creates directory

### PR DESCRIPTION
**Pull request by personjs**
Check for any existing system object in docker script to clear warning message (Resolving PR comments)
[NO NEW TESTS NEEDED]
Signed-off-by: Stephen Person <stephen.person.12@cnu.edu>

#### What this PR does / why we need it:
The `echo` message in docker cli script does not specify whether to create a directory or file, so I've added an extra check if the directory /etc/containers/nodocker exists.

#### How to verify it

Create /etc/containers/nodocker directory,  run `docker`, and verify warning message is not displayed.

#### Which issue(s) this PR fixes:

None

### CODE REVIEW

File: docker [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
-[ -f /etc/containers/nodocker ] || \
+[[ -f /etc/containers/nodocker || -d /etc/containers/nodocker ]] || \
```
Comment (rhatdan) [inline docker:2]: Why not just change the -f to -e, then if there is any file system object called /etc/containers/nodocker it will not show the message.

**Comment by rhatdan** [comment]
You need to sign your PRs
git commit -a --amend -s
git push --force

File: docker (hunk 1 above)
Comment (personjs) [reply docker]: Good call, I've made the change.

**Comment by rhatdan** [comment]
Could you merge your PRs and move the [NO NEW TESTS NEEDED] line to a separate line with whitespace beween it and the first line.  Basically the third line.

git rebase -i origin
git push --force
LGTM
/approve
@containers/podman-maintainers PTAL

**Comment by personjs** [comment]
>

**Comment by personjs** [comment]
That (close) was an accident. Should be good now. Thanks!

**Comment by rhatdan** [comment]
The signature has to be on a line by itself.

**Review by TomSweeneyRedHat — COMMENTED** [review]
LGTM
once tests are happy

**Comment by rhatdan** [comment]
[NO TESTS NEEDED]
Should be
[NO NEW TESTS NEEDED]

**Comment by umohnani8** [comment]
LGTM once commit message is amended

**Pull request closed without merging**